src/lib/transactionBuilder.js

```diff
@@ -736,6 +736,137 @@ export default class TransactionBuilder {
         this.tronWeb.fullNode.request('wallet/triggersmartcontract', args, 'post').then(transaction => resultManager(transaction, callback)).catch(err => callback(err));
     }
 
+    triggerConstantContract(...params) {
+        if (typeof params[2] !== 'object') {
+            params[2] = {
+                feeLimit: params[2]
+            }
+            params.splice(3, 1)
+        }
+        return this._triggerConstantContract(...params);
+    }
```

Comment:
```
triggerConstantContract(...params) {
    params[2]._isConstant = true
    return this._triggerSmartContract(...params);
}
```

Reply:
@sullof Has fixed, please review when you have time.